server/system_command.c

```diff
@@ -549,7 +549,7 @@ static error__t process_system_put(
 
 
 static error__t process_system_put_table(
-    const char *command, bool append, bool binary,
+    const char *command, bool append, bool binary, bool more_expected,
```

Comment:
Oh!  I've just noticed a troubling discrepancy: 
https://github.com/PandABlocks/PandABlocks-server/blob/bd16e9011d589aeb977c909c69c38a0ccc2d121b/server/config_server.c#L279
https://github.com/PandABlocks/PandABlocks-server/blob/bd16e9011d589aeb977c909c69c38a0ccc2d121b/server/config_server.h#L151-L153
https://github.com/PandABlocks/PandABlocks-server/blob/bd16e9011d589aeb977c909c69c38a0ccc2d121b/server/system_command.c#L551-L553
It's unfortunate that we have three boolean arguments in a row, but that can't really be helped.  The problem here is two-fold:
1. Should the extra parameter be called `more_expected` or `last_table` (these have opposite meanings!)
2. Are we defining our parameters in the correct order everywhere?

This needs a careful check of all affected files!

Reply:
I've had a look through all the instances of `.put_table` and it looks like the only error is in the declaration of `process_system_put_table`, so that's an easy fix!

Reply:
Thanks a lot, I applied your patch